### ISSUE TITLE
Inference mode not working as expected

### ISSUE THREAD

**Issue by mvish7** [Jieqianyu/PANet#1]
Hello,

Thanks for making the code available. 

I was trying to test the model by using  the following command. 
`python cfg_train.py  --config cfgs/panet.yaml --onlytest --pretrained_ckpt weights/kitti_backbone_v2.pth` or 
`python cfg_train.py  --config cfgs/panet.yaml --onlytest --pretrained_ckpt weights/kitti_panet.pth`

The script works and saves the predictions under .label file for each scan and each sequence. When I tried to visualize the predictions I found that semantic and/or instance predictions are off.
 Please find attached snapshot from SemanticKITTI API.  Please not that the limits from `cfgs/panet.yaml` were also applied during the visualization. 
![Screenshot from 2023-12-06 18-18-39](https://github.com/Jieqianyu/PANet/assets/44236845/17affd3f-f7e0-4d62-bbc6-ceb2bddc1256)

Upon debugging the code I found that function [gen_proposals](https://github.com/Jieqianyu/PANet/blob/cb696712abfe87bccf8195dbb5ea5b5df0885a8c/models/model.py#L190) in` model.py` returns `None`. 

>   def gen_proposals(self, embeddings, sem_logits, ins_fea, batch_idx, gt_centers=None, is_test=True):
>         #device = embeddings.device
>         pc_range = torch.tensor(self.pc_range, device=device)
>         valid_classes = torch.tensor(sorted(valid_xentropy_ids), device=device).long()
>         sem_preds = sem_logits.argmax(1) + self.plus if is_test else sem_logits
>         cluster_inds_list, embeddings_list, feats_list, valid_masks_list = [], [], [], []
>         valid = (sem_preds.reshape(-1, 1) == valid_classes).any(dim=-1)
>         if valid.sum() == 0:
>             return None

The reason behind that seems to be on line `valid = (sem_preds.reshape(-1, 1) == valid_classes).any(dim=-1)` as no valid segmentation classes are found. 

Could you please confirm the behavior at your end? If I'm doing something wrong (most likely), could you provide hints to reproduce the results from the  paper?

**Comment by Jieqianyu** [comment]
Did you put both ``kitti_backbone_v2.pth`` and ``kitti_panet.pth``  in the ``weights`` folder? It looks like the semantic results are also empty.

**Comment by Jieqianyu** [comment]
<img width="784" alt="image" src="https://github.com/Jieqianyu/PANet/assets/39148698/7d4ec1e9-e821-41b5-9596-16d2960848d0">

**Comment by Jieqianyu** [comment]
We were able to get the correct visualization in our machine. Maybe you can check the weights and running environment, especially the version of spconv.

**Comment by Jieqianyu** [comment]
![image](https://github.com/Jieqianyu/PANet/assets/39148698/f3bcb68b-62d2-4dd3-9db7-231c694f3927)

**Comment by Jieqianyu** [comment]
We hope this helps you. If you have any questions, please feel free to ask us.

**Comment by mvish7** [comment]
Thank you for your response. I got the inference mode working after installing spconv1.1 from source.
